acceptance/tests/facts/partitions.rb

```diff
@@ -13,7 +13,7 @@
       ['uuid', /^[-a-zA-Z0-9]+$/],
       ['partuuid', /^[-a-f0-9]+$/],
       ['mount', /^\/.*/],
-      ['label', /\w+/],
```

Comment:
The problem is that for this array of entries, they may or may not bet set depending on the different operating systems and how they are configured. So rather adding a bunch of do this for this operating system, we instead if the label is set then the regex should match it